test/unit/test_requote.js

```diff
@@ -0,0 +1,227 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of ThingEngine
+//
+// Copyright 2020 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Mehrad Moradshahi <mehrad@cs.stanford.edu>
+//
+// See COPYING for details
+"use strict";
+
+const assert = require('assert');
+
+const { requoteSentence } = require('../../tool/requote');
+
+const SINGLE_TURN_TEST_CASES = [
```

Comment:
If you write this way, it's not very readable.
It should be (input, output) pairs.

The point of unit tests is to test specific cases and specific behavior. We already have the command-line tests for general functional tests (no crash, no regressions). I can't even look at these tests to see what they are testing, let alone see if they are correct.